### ISSUE TITLE
[1.10.2] Server crash on startup due to ticking entity

### ISSUE THREAD

**Issue by themechman** [blay09/NetherPortalFix#18]
Unfortunately I've located another problem with 3.3.7 for 1.10.2

Upon adding this version to my personal modpack's server it crashes it with a ticking entity. It lists the location and offending mod, however when I make an attempt to remove the ticking block in MCedit, there is literally nothing at the location referenced. The location it references changes every time the server is started as well.

I remove NPF 3.3.7 and restart the server without crash or fault.

[crash-2017-10-19_12.57.45-server.txt](https://github.com/blay09/NetherPortalFix/files/1399347/crash-2017-10-19_12.57.45-server.txt)
[crash-2017-10-19_12.50.41-server.txt](https://github.com/blay09/NetherPortalFix/files/1399351/crash-2017-10-19_12.50.41-server.txt)

**Comment by BlayTheNinth** [comment]
This appears to be an issue on McMultipart's side, there's several reports of similar crashes with other mods, so far with no solution:

https://github.com/amadornes/MCMultiPart/issues/71
https://github.com/amadornes/MCMultiPart/issues/81
https://github.com/aidancbrady/Mekanism/issues/4451

Does the crash happen immediately or at random? If it's reproducible I can go in and try to do some debugging myself.

**Comment by themechman** [comment]
I added 3.3.7 initially and the world loaded fine. I had to restart server because of power loss enforced shutdown. After this, it hasnt work right. Shutdown was safely done as I did it by hand.

To answer your question, it happens once overworld dim is initialized (See log).

Two dims have 5x5 chunk loaders (0 and -28). If I remove the chunk loading mod (Persistent Bits 1.0.6), the problem also resolves itself. However this shuts everything down.

I am including the last forge log I have where it crashed. Thankfully you dont have to dig too much as it's at the very end, but about 12:57:45 is where TickDynamic freaks out. I will pull this mod out next and see if that is the culprit.

[fml-server-3.log](https://github.com/blay09/NetherPortalFix/files/1400436/fml-server-3.log)

Also of note. Aidan wont work on 1.10.x Mek. He's already refused to fix a power glitch which makes his machines run with zero power. It was fixed in 1.11, but not 1.10
